test/gen-server/migrations.ts

```diff
@@ -49,7 +51,7 @@ const migrations = [Initial, Login, PinDocs, UserPicture, DisplayEmail, DisplayE
                     CustomerIndex, ExtraIndexes, OrgHost, DocRemovedAt, Prefs,
                     ExternalBilling, DocOptions, Secret, UserOptions, GracePeriodStart,
                     DocumentUsage, Activations, UserConnectId, UserUUID, UserUniqueRefUUID,
-                    Forks, ForkIndexes, ActivationPrefs, AssistantLimit, Shares];
+                    Forks, ForkIndexes, ActivationPrefs, AssistantLimit, Shares, UserLastConnection];
```

Comment:
Tests fails with error `no such column: users.last_connection_at` triggered when run migration `UserUUID`
because it uses the entity `User` that has no `last_connection_at` for the moment (the migration come later) : [see here ](https://github.com/gristlabs/grist-core/blob/4567fad94787c20f65db68e744c47d5f44b932e4/app/gen-server/migration/1663851423064-UserUUID.ts#L20)
Locally when I put the migration `UserLastConnection` before `UserUUID` it works but I think it's not a good fixe. 
What about duplicate the entity inside the migration has it have to be at this point ? (Not convince about this approach)

Reply:
We'll need to update `UserUUID` and any other affected migrations to not depend on the `User` entity. I don't know of any workarounds, unfortunately.

Reply:
How do you imagine updating `UserUUID` to not depend ont the `User` entity has it exists to modify this entity ?

Reply:
Replacing the methods called on the `User` entity with equivalent raw SQL is one idea. (typeorm supports executing raw SQL via the `query` method.)

The `chunk` option could be replaced with a for loop that does the `UPDATE` calls in chunks.